host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@10.0.1.2_cuda.cmake

```diff
@@ -109,7 +109,7 @@ set(MFEM_DIR "${TPL_ROOT}/mfem-4.5.0" CACHE PATH "")
 
 set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.22" CACHE PATH "")
 
-set(LUA_DIR "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2022_12_08_21_33_38/clang-10.0.1.1/lua-5.4.4" CACHE PATH "")
+set(LUA_DIR "${TPL_ROOT}/lua-5.4.4" CACHE PATH "")
```

Comment:
Note: The new uberenv gets spack to respect our compiler spec!
Thanks @white238 for working through this!